pkg/trace/testutil/otlp.go

```diff
@@ -29,6 +30,15 @@ type OTLPSpanEvent struct {
 	Dropped    uint32                 `json:"dropped_attributes_count"`
 }
 
+// OTLPSpanLink defines an OTLP test span link.
+type OTLPSpanLink struct {
+	TraceID    string                 `json:"trace_id"`
```

Comment:
Unrelated to the code review: can a span be linked to a span in another trace?

Reply:
Yes they can - in fact, that's probably the more common case.

Reply:
Wow. Interesting.